### DISCUSSION TITLE
Switch to the 'Style Picker tool' using 'Alt' with the filler tool as optional.

### DISCUSSION THREAD

**Discussion by F-Burning** [opentoonz/opentoonz#6791] [Ideas]
**Is your feature request related to a problem? Please describe**

My problem is that I am used to using 'Alt' with all painting programs to select a color, but in Opentoonz 'Alt' is reserved for other options.

**Describe the solution you'd like**

The possibility of being able to toggle the 'Style Picker tool' while using the fill tool (only would work with this one) by pressing the 'Alt' key as an option in '''Preferences'

**Comment by RodneyBaker** [comment]
If I read this correctly the issue here is that Alt by itself is not an option as a shortcut key.
Alt + another key can be used.

Currently in Opentoonz we can use multiple keys so perhaps Style Picker would be Alt + I and Fill Tool Alt + F (or as desired).

What are we missing here?
